app/v3/models/domain/LossClaim.scala

```diff
@@ -16,38 +16,38 @@
 
 package v3.models.domain
 
-import play.api.libs.json.{Json, OWrites, Reads}
+import play.api.libs.json._
 import v3.models.requestData.DownstreamTaxYear
 
-case class LossClaim(taxYear: String,
-                     typeOfLoss: TypeOfLoss,
-                     typeOfClaim: TypeOfClaim,
-                     businessId: Option[String])
+case class LossClaim(taxYearClaimedFor: String, typeOfLoss: TypeOfLoss, typeOfClaim: TypeOfClaim, businessId: String)
+
 object LossClaim {
   implicit val reads: Reads[LossClaim] = Json.reads[LossClaim]
 
-  implicit val writes: OWrites[LossClaim] = (claim: LossClaim) => {
-    val jsObject = (claim.typeOfLoss.isUkProperty, claim.typeOfLoss.isForeignProperty) match {
-      case (true,_) =>
+  implicit val writes: OWrites[LossClaim] = (lossClaim: LossClaim) => {
+    (lossClaim.typeOfLoss) match {
+      case TypeOfLoss.`uk-property-non-fhl` =>
         Json.obj(
-          "incomeSourceType" -> claim.typeOfLoss.toIncomeSourceType,
-          "reliefClaimed" -> claim.typeOfClaim.toReliefClaimed,
-          "taxYear" -> DownstreamTaxYear.fromMtd(claim.taxYear).toString
+          "taxYear"          -> DownstreamTaxYear.fromMtd(lossClaim.taxYearClaimedFor).value,
+          "incomeSourceType" -> TypeOfLoss.`uk-property-non-fhl`.toIncomeSourceType.get,
```

Comment:
Since you're using separate cases anyway (and `toIncomeSourceType` doesn't work for self employment), why not just specify the the income source type explicitly here? (This is probably a sign that we shouldn't really be using the same TypeOfLoss for BF losses _and_ claims. But maybe we can think about that later.)
```suggestion
          "incomeSourceType" -> "02",
```
Likewise for `foreign-property` case.

Up to you.